### PR TITLE
show field type when editing a field

### PR DESCRIPTION
**Pull request by ypresion**
This pull request makes the following changes:
- 'Edit field' in surveys now shows the type of the field 
d9f145bc5bc2a1a1d55b8e8ecc32aee6

Testing checklist:
- [ ] log into your deployment
- [ ] go to survey
- [ ] select a field to edit
- [ ] modal's title should be in format '**Edit** *field type* **field**'

- [x] I certify that I ran my checklist

[Fixes # 1555 in ushahidi/platform](https://github.com/ushahidi/platform/issues/1555)

Ping @ushahidi/platform

### CODE REVIEW

**Comment by ypresion** [comment]
[clabot:check]

File: app/settings/surveys/survey-editor.directive.js [hunk 1]
```diff
@@ -472,10 +472,61 @@ function SurveyEditorController(
             ModalService.close();
         }
         $scope.editAttribute = attribute;
-        var title = attribute.id ? 'survey.edit_field' : 'survey.add_field';
+        var fieldType = getFieldType(attribute) ? getFieldType(attribute) : attribute.label;
+        var title = attribute.id ? $translate.instant('survey.edit_field', {fieldType: fieldType}) : 'survey.add_field';
```
Comment (Angamanga) [inline app/settings/surveys/survey-editor.directive.js:476]: Could you add the type of field when adding a new field as well? Same method as when editing, just add the translate-values to "survey.add_field" as well.

File: app/settings/surveys/survey-editor.directive.js [hunk 2]
```diff
@@ -472,10 +472,61 @@ function SurveyEditorController(
...
+        title = title.replace('&amp;', '&');
         ModalService.openTemplate('<survey-attribute-editor></survey-attribute-editor>', title, '', $scope, true, true);
     }
 
+    function getFieldType(attribute) {
```
Comment (Angamanga) [inline app/settings/surveys/survey-editor.directive.js:481]: Neat solution!

**Review by Angamanga — CHANGES_REQUESTED** [review]
Well done! I just left a small change in the comments, otherwise it works fine!

**Comment by Angamanga** [comment]
@ypresion Do you plan to fix the requested changes for this pr? Otherwise I'll merge it into its own branch so someone else can continue 👍

**Comment by Angamanga** [comment]
Closing this in favour of https://github.com/ushahidi/platform-client/pull/1630

**Pull request closed without merging**